css/properties/background-clip.json

```diff
@@ -118,7 +118,7 @@
                 "version_added": "12"
               },
               "firefox": {
-                "version_added": "≤16"
+                "version_added": "4"
```

Comment:
This aligns with the `partial_implementation` note on the parent feature 👍

This _could_ be restructured as an alt. name of the two subfeatures, but that makes it less obvious that it's connected to the prefixed implementation.